desktop/apps/editorial_features/index.coffee

```diff
@@ -1,10 +1,32 @@
 express = require 'express'
 routes = require './routes'
+httpProxy = require 'http-proxy'
+proxy = httpProxy.createProxyServer(changeOrigin: true, ignorePath: true)
+{ WHITELISTED_VANITY_ASSETS, VANITY_BUCKET } = require '../../config.coffee'
 { crop } = require '../../components/resizer/index'
+adminOnly = require '../../lib/admin_only'
 
 app = module.exports = express()
 app.set 'views', "#{__dirname}"
 app.set 'view engine', 'jade'
 app.locals.crop = crop
 
 app.get '/2016-year-in-art', routes.eoy
+
+app.get '/2017-venice-biennale/jw', adminOnly, (req, res, next) ->
+  res.render 'components/venice_2017/templates/jw'
+
+app.get '/2017-venice-biennale/googlevr', adminOnly, (req, res, next) ->
+  res.render 'components/venice_2017/templates/googlevr'
+
+app.get '/2017-venice-biennale/aframe', adminOnly, (req, res, next) ->
+  res.render 'components/venice_2017/templates/aframe'
+
+app.get '/vanity/*', (req, res, next) ->
+  whitelistedAssets = WHITELISTED_VANITY_ASSETS.split(',')
+  return next() unless req.params[0] in whitelistedAssets
+  req.headers['host'] = VANITY_BUCKET
+  proxy.on 'error', (err) ->
```

Comment:
Since `proxy` is declared above this route handler scope, I wonder if this risks concurrency bugs where a separate request fails and causes all open requests to redirect to 301. It would probably be more appropriate to use `proxy.web`'s callback API e.g. `proxy.web req, res, target: target, (err) -> res.redirect 301, '/articles' if err`